src/app/components/filter/filters/tabbed-value-entry.jsx

```diff
@@ -98,7 +98,7 @@ function ChplTabbedValueEntry(props) {
 
   return (
     <>
-      <AppBar className={classes.appBar} position="sticky">
+      <AppBar className={classes.appBar} elevation={0} position="sticky">
```

Comment:
What does this do? I can't find the attribute in the docs for AppBar

Reply:
takes away the default drop shadow that comes with an appbar